src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -18,6 +18,7 @@
 using namespace ::Microsoft::Console::Types;
 using namespace ::Microsoft::Terminal::Core;
 using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Input;

```

Comment:
Do we need this?

Reply:
I would not have added it if we did not need it :grin: